lib/jellyfish/room.ex

```diff
@@ -72,6 +72,11 @@ defmodule Jellyfish.Room do
     end
   end
 
+  @spec get_forwarded_tracks_in_room(id()) :: :ok
```

Comment:
```suggestion
  @spec request_forwarded_tracks_in_room(id()) :: :ok
```

Maybe change the name somehow, so it doesn't sound like this functions returns the forwarded tracks?